1-JavaScript/Week3/homework/ex4-shoppingCart.js

```diff
@@ -26,6 +26,13 @@ function addToShoppingCart(/* parameters go here */) {
 }
 
 // ! Test functions (plain vanilla JavaScript)
+function test0() {
+  console.log('Test 0: addShoppingCart() with intitial array of length 2 should not throw an index out of bounds exception');
+  const expected = 'You bought bananas, milk!';
+  const actual = addToShoppingCart(null);
```

Comment:
Perhaps the concept of using `null` to explicitly indicate _no data_ does not entirely gel yet in this first week of JavaScript homework. Suggest to show the function being called without an argument.

We should also update the assignment description to explicitly indicate what should happen if the function is called without an argument.

```suggestion
  const actual = addToShoppingCart();
```